Lookuptable/testPages/applications/application_page.py

```diff
@@ -0,0 +1,217 @@
+import time
+
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.by import By
+
+from common_utilities.selenium.base_page import BasePage
+from common_utilities.generate_random_string import fetch_random_string
+from HQSmokeTests.userInputs.user_inputs import UserData
+from common_utilities.path_settings import PathSettings
+from HQSmokeTests.testPages.users.org_structure_page import latest_download_file
+
+""""Contains test page elements and functions related to the applications on Commcare"""
+
+
+class ApplicationPage(BasePage):
```

Comment:
I don't think there's any need to add redundant classes when we have existing classes here https://github.com/dimagi/dimagi-qa/tree/master/HQSmokeTests/testPages/applications

we can simply create objects for the existing classes to call the required functions.